client/src/pages/proposalDetails/ProposalDetails.css

```diff
@@ -0,0 +1,117 @@
+:root {
+    --green: #96AD00;
+    --mutedGreen: #D9DEBC;
+    --lightGrey: #F3F3F3;
+    --black: #353739;
+    --headingFont: 'IBM Plex Sans', sans-serif;
+    --bodyFont: 'Lexend Deca';
+    --descriptionAndProgressBarsWidth: 87%;
```

Comment:
This seems weird to define as a var in the root if it'll only be used in one screen/page

Reply:
It would also be good to move the general variables like `--green` and `--mutedGreen` to a more overarching file like "App.css". It's effectively the same but more semantic!